lib/Parse/ParseStmt.cpp

```diff
@@ -1103,7 +1103,47 @@ Parser::parseAvailabilitySpecList(SmallVectorImpl<AvailabilitySpec *> &Specs) {
       consumeToken();
       Status.setIsParseError();
     } else if (consumeIf(tok::comma)) {
-      // keep going.
```

Comment:
In general, can you run git-clang-format on this patch.  There's like 4 or 5 places with some interesting indentation.